src/stories/Toolbar.stories.tsx

```diff
@@ -39,15 +39,7 @@ const Template: Story<{ narrow?: boolean }> = ({ narrow }) => {
 
   return (
     <>
-      <div
-        style={{
-          display: 'flex',
-          maxWidth: narrow ? '30rem' : undefined,
-          height: '2.5rem',
-          marginLeft: 'auto',
-          backgroundColor: 'var(--primary-bg)',
-        }}
-      >
+      <div style={narrow ? { maxWidth: '30rem', marginLeft: 'auto' } : {}}>
```

Comment:
The toolbar no longer needs a wrapper element. (This one is now just for the responsive toolbar story).